libr/core/visual.c

```diff
@@ -914,18 +914,23 @@ R_API int r_core_visual_xrefs_X(RCore *core) {
 		addr += core->print->cur;
 	}
 
+	r_cons_clear00 ();
+	r_cons_gotoxy (1, 1);
+	r_cons_printf ("[GOTO REF]> \n");
+
+	ut64 references[10];
```

Comment:
better define this at the begining of the function, and if you do so initialize with = {0};